build/.docker/stack.yaml

```diff
@@ -22,6 +22,10 @@ services:
       ME_CONFIG_MONGODB_SERVER: mongodb
   application:
     container_name: onboarding-go
-    build: ../
+    ports:
+      - 8080:8080
+    build:
+      context: ../../
+      dockerfile: build/.docker/Dockerfile
     depends_on:
       - mongodb
```

Comment:
Docker compose should override the default entrypoint to allow remote debugging.

Reply:
Do you have any tips to do it?